### ISSUE TITLE
Wsparcie dla prac dyplomowych na studiach podyplomowych

### ISSUE THREAD

**Issue by turbaszek** [lchmiel/SGGW-Thesis-LaTeX#6]
Obecnie szablon wspiera możliwość wyboru pracy licencjackiej lub magisterskiej. Obecnie pisze pracę podyplomową i chętnie skorzystałbym z tego szablonu.

@lchmiel  czy wystarczy jeśli zostanie dodany owy plik `.png` pozbawiony dopisku `Licencjacka` lub `Magisterska`?
<img width="611" alt="Screenshot 2022-01-23 at 12 50 39" src="https://user-images.githubusercontent.com/9528307/150676979-b97e6a6c-2c59-4344-bb79-7466f7b40987.png">

**Comment by lchmiel** [comment]
Oczywiście, może Pan modyfikować te obrazy na swój użytek. Proszę jeszcze zwrócić uwagę na makro \thesis{} z parametrem, w którym się wpisuje tekst {Praca dyplomowa na kierunku:}. Może tu należałoby napisać \thesis{Praca podyplomowa na kierunku:} lub podobnie. 

Jeśli prace na studiach podyplomowych mają swój wzorzec, to można by dodać opcje lub flagi do klasy. Nie interesowałęm się tym tematem, ale jeśli jest potrzeba, to czemu nie.

Pozdrawiam
Leszek Chmielewski

**Comment by turbaszek** [comment]
> Jeśli prace na studiach podyplomowych mają swój wzorzec, to można by dodać opcje lub flagi do klasy. Nie interesowałem się tym tematem, ale jeśli jest potrzeba, to czemu nie.

Też o tym pomyślałem, natomiast wymagania dotyczące prac podyplomowych które znalazłem wydaję się być bardziej okrojone. Jeśli pojawią się jakieś uwagi podczas recenzji to rozważę dodanie poprawek.
